crates/ruff_server/src/server/api/requests/code_action.rs

```diff
@@ -105,7 +105,7 @@ fn fix_all(snapshot: &DocumentSnapshot) -> crate::Result<CodeActionOrCommand> {
                 document,
                 snapshot.resolved_client_capabilities(),
                 snapshot.url(),
-                &snapshot.configuration().linter,
+                &snapshot.ruff_settings().linter,
```

Comment:
nit: same as above to drop the "ruff" here to make it `&snapshot.settings().linter`